mcs/class/System.Net.Http/CFContentStream.cs

```diff
@@ -1,156 +0,0 @@
-//
```

Comment:
these are used from the xamarin-macios repo: https://github.com/xamarin/xamarin-macios/blob/2e5ee3e90a73d8421930cde517e1dce9bd0ee506/src/Makefile#L253-L255